Loading expert policy from... aai4160/policies/experts/Ant.pkl
obs (1, 111) (1, 111)
Done restoring expert policy...
########################
logging outputs to  /root/RL-HW/HW1/aai4160/scripts/../../data/q1_bc_ant_Ant-v4_31-03-2024_05-57-05
########################
Using GPU id 0
Traceback (most recent call last):
  File "aai4160/scripts/run_hw1.py", line 155, in <module>
    main()
  File "aai4160/scripts/run_hw1.py", line 151, in main
    run_bc(params)
  File "aai4160/scripts/run_hw1.py", line 65, in run_bc
    trainer = BCTrainer(params)
  File "/root/RL-HW/HW1/aai4160/infrastructure/bc_trainer.py", line 88, in __init__
    self.env = gym.make(self.params['env_name'], **self.params['env_kwargs'])
  File "/root/miniconda3/envs/rl/lib/python3.8/site-packages/gym/envs/registration.py", line 619, in make
    env_creator = load(spec_.entry_point)
  File "/root/miniconda3/envs/rl/lib/python3.8/site-packages/gym/envs/registration.py", line 62, in load
    mod = importlib.import_module(mod_name)
  File "/root/miniconda3/envs/rl/lib/python3.8/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
  File "<frozen importlib._bootstrap>", line 991, in _find_and_load
  File "<frozen importlib._bootstrap>", line 961, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
  File "<frozen importlib._bootstrap>", line 991, in _find_and_load
  File "<frozen importlib._bootstrap>", line 975, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 671, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/root/miniconda3/envs/rl/lib/python3.8/site-packages/gym/envs/mujoco/__init__.py", line 1, in <module>
    from gym.envs.mujoco.mujoco_env import MujocoEnv, MuJocoPyEnv  # isort:skip
  File "/root/miniconda3/envs/rl/lib/python3.8/site-packages/gym/envs/mujoco/mujoco_env.py", line 22, in <module>
    import mujoco
  File "/root/miniconda3/envs/rl/lib/python3.8/site-packages/mujoco/__init__.py", line 50, in <module>
    from mujoco.osmesa import GLContext
  File "/root/miniconda3/envs/rl/lib/python3.8/site-packages/mujoco/osmesa/__init__.py", line 31, in <module>
    from OpenGL import GL
  File "/root/miniconda3/envs/rl/lib/python3.8/site-packages/OpenGL/GL/__init__.py", line 4, in <module>
    from OpenGL.GL.VERSION.GL_1_1 import *
  File "/root/miniconda3/envs/rl/lib/python3.8/site-packages/OpenGL/GL/VERSION/GL_1_1.py", line 14, in <module>
    from OpenGL.raw.GL.VERSION.GL_1_1 import *
  File "/root/miniconda3/envs/rl/lib/python3.8/site-packages/OpenGL/raw/GL/VERSION/GL_1_1.py", line 7, in <module>
    from OpenGL.raw.GL import _errors
  File "/root/miniconda3/envs/rl/lib/python3.8/site-packages/OpenGL/raw/GL/_errors.py", line 4, in <module>
    _error_checker = _ErrorChecker( _p, _p.GL.glGetError )
AttributeError: 'NoneType' object has no attribute 'glGetError'